src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountForSixDays = 20;
+  const discountForSevenDays = 50;
```

Comment:
do not use numbers in variables
what if wi will change daysForLargeDiscount to 10? Rename variable?